### DISCUSSION TITLE
Does the icon name support uppercase letters in  preset-icons plugin?

### DISCUSSION THREAD

**Discussion by shenjunjian** [unocss/unocss#4152] [Polls]
details:  https://github.com/unocss/unocss/pull/4149

@zzyv says:  "We keep the lowercase syntax in all regular expressions by default,"  and  then close my pr. 
but I don't agree.

I think: 
- the DOM's className supports uppercase characters
- @iconify/tools  create IconSet also supports uppercase characters
- for some technical terms, uppercase characters are necessary, such as ECS, IBM, KFC, Apple,

I don't have "lowercase syntax" obsessive-compulsive disorder, and I think it's natural to have capital letters.

-------
preset-icons plugin 是否应该支持大写图标名？
@zzyv says:  "我们要保持用小写语法在所有的正则中"  ,之后关闭了我的pr. 

但我不同意这个说法，我认为：
- DOM 的className原生支持大写
- @iconify/tools 包中的IconSet中，图标名也支持大写
- 在一些专业术语中，大写字母很常见  比如： ECS, IBM, KFC, Apple

我没有 lowercase syntax 的强迫症，我认为有大写字母是理所应当的

**Comment by zyyv** [comment]
@shenjunjian  I'm sorry for closing your PR in a hurry, and i reopen it. We will discuss supporting uppercase syntax and wait for your final review from antfu. This discussion can get more people's opinions.

Personally, I don't care about the support for uppercase and lowercase. I personally like to be consistent. I selected option 2.
